react/week1/todo-app/src/App.css

```diff
@@ -0,0 +1,70 @@
+.App {
```

Comment:
please try to have the same structure for your classes and ids. try to use lowercase so replace "App" to "app".